framework/src/modules/chain/blocks/process.js

```diff
@@ -188,7 +169,7 @@ class BlocksProcess {
 				continue;
 			}
 			// eslint-disable-next-line no-await-in-loop
-			lastBlock = await this.applyBlock(block, lastBlock);
+			lastBlock = await this.processBlock(block, lastBlock);
```

Comment:
this should be apply. block should not be saved

Reply:
I'll revert this change.